buildSrc/src/main/java/Config.kt

```diff
@@ -40,12 +40,25 @@ object Config {
 
     object QualityPlugins {
         val jacocoVersion = "0.8.5"
+        val spotless = "com.diffplug.gradle.spotless"
         val spotlessVersion = "3.25.0"
         val errorpronePlugin = "net.ltgt.gradle:gradle-errorprone-plugin:1.1.1"
     }
 
     object Sentry {
         val SENTRY_CLIENT_NAME = "sentry.java.android"
+        val group = "io.sentry"
+        val version = "2.0.0-alpha01"
+        val description = "SDK for sentry.io"
+        val buildVersionCode = 20001
+        val website = "https://sentry.io"
+        val userOrg = "getsentry"
+        val repoName = "sentry-android"
+        val licence = "MIT"
+        val issueTracker = "https://github.com/getsentry/sentry-android/issues"
+        val repository = "https://github.com/getsentry/sentry-android"
+        val autoPublish = false // This boolean defines if the package will be published when uploaded. If this is false, the package will still be uploaded to bintray but you'll have to publish it manually.
```

Comment:
@bruno-garcia what do you think? true or false? if false, we need to go on the website and press buttons

Reply:
I'd just auto publish. It's important not to have the keys sourced to the shell automatically. (which means not using local.properties).
Let's discuss this offline